readme.md

```diff
@@ -9,6 +9,13 @@ Web based viewer for servers and bots
 [![Irc](https://img.shields.io/badge/chat-on%20irc-brightgreen.svg)](https://irc.gitter.im/)
 
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-viewer)
+![npm](https://img.shields.io/npm/dt/mineflayer-pathfinder)
```

Comment:
I don't know if the download could really bring anything, there are enough badges